code/controllers/subsystem/instancing.dm

```diff
@@ -43,17 +43,19 @@ SUBSYSTEM_DEF(instancing)
 	var/ckey_json = json_encode(ckeys)
 
 	// Yes I care about performance savings this much here to mass execute this shit
-	var/list/datum/db_query/queries = list()
-	queries += SSdbcore.NewQuery("UPDATE instance_data_cache SET key_value=:json WHERE key_name='playerlist' AND server_id=:sid", list(
+	var/datum/db_query/dbq = SSdbcore.NewQuery("UPDATE instance_data_cache SET key_value=:json WHERE key_name='playerlist' AND server_id=:sid", list(
 		"json" = ckey_json,
 		"sid" = GLOB.configuration.system.instance_id
 	))
-	queries += SSdbcore.NewQuery("UPDATE instance_data_cache SET key_value=:count WHERE key_name='playercount' AND server_id=:sid", list(
+	dbq.warn_execute(FALSE, FALSE)
+	qdel(dbq)
+
+	dbq = SSdbcore.NewQuery("UPDATE instance_data_cache SET key_value=:count WHERE key_name='playercount' AND server_id=:sid", list(
```

Comment:
Nit: making this dbq2 instead of re-assigning a qdeleted is more common for style here